corehq/blobs/migrate.py

```diff
@@ -501,16 +513,20 @@ def migrate(self, filename=None, reset=False, max_retry=2, chunk_size=100):
         xform.SubmissionErrorLog,
         ("HQSubmission", xform.XFormInstance),
     ], BlobDbBackendExporter),
-    ExportByDomainSQL("export_domain_sql_xform", [
-        xform.XFormInstance,
-        ("XFormInstance-Deleted", xform.XFormInstance),
-        xform.XFormArchived,
-        xform.XFormDeprecated,
-        xform.XFormDuplicate,
-        xform.XFormError,
-        xform.SubmissionErrorLog,
-        ("HQSubmission", xform.XFormInstance),
-    ], SqlFormAttachmentExporter),
+    SqlModelMigrator(
+        "sql_xforms",
+        sql_xform.XFormAttachmentSQL,
+        SqlFormAttachmentExporter
+    ),
+    ExportByDomain("saved_exports", [
```

Comment:
I think I remember seeing this and restores and it looked like they were only the cached results? Or is this something different than the cache?